packages/rancher-logging/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -14,11 +14,20 @@
 +{{- end -}}
 +
 +{{- define "windowsEnabled" }}
-+  {{- if not (kindIs "invalid" .Values.global.cattle.windows) }}
-+  {{- if not (kindIs "invalid" .Values.global.cattle.windows.enabled) }}
-+  {{- if .Values.global.cattle.windows.enabled }}
++{{- if not (kindIs "invalid" .Values.global.cattle.windows) }}
++{{- if not (kindIs "invalid" .Values.global.cattle.windows.enabled) }}
++{{- if .Values.global.cattle.windows.enabled }}
 +true
-+  {{- end }}
-+  {{- end }}
-+  {{- end }}
++{{- end }}
++{{- end }}
++{{- end }}
++{{- end }}
++
++{{- define "windowsPathPrefix" }}
++{{- $temp := (default "c:\\" .Values.global.cattle.rkeWindowsPathPrefix | replace "\\" "/" | replace "//" "/") }}
```

Comment:
Can we rename this to `$prefixPath`?

Reply:
Not sure if I like that idea actually since the RKE Linux prefix path can be different.